src/backend/patches/hdr-histogram-js+2.0.3.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/node_modules/hdr-histogram-js/dist/formatters.js b/node_modules/hdr-histogram-js/dist/formatters.js
+index 7db1558..3003988 100644
+--- a/node_modules/hdr-histogram-js/dist/formatters.js
++++ b/node_modules/hdr-histogram-js/dist/formatters.js
+@@ -25,12 +25,7 @@ exports.keepSignificantDigits = (digits) => (value) => {
+     return value;
+ };
+ exports.floatFormatter = (size, fractionDigits) => {
+-    const numberFormatter = new Intl.NumberFormat("en-US", {
+-        maximumFractionDigits: fractionDigits,
+-        minimumFractionDigits: fractionDigits,
+-        useGrouping: false,
+-    });
```

Comment:
for context, the v8 engine we use doesn't include the `Intl` apis, so this fails on app startup

Reply:
That's frustrating! Another solution would have been to define a polyfill somewhere like

```js
global.Intl = { NumberFormat: () => (num) => num + '' }
```

But I think your solution is probably the best in this case.